definitions/npm/bluebird_v3.x.x/flow_v0.104.x-/test_bluebird_v3.x.x.js

```diff
@@ -51,7 +51,7 @@ Bluebird.all([1, Bluebird.resolve(1), Promise.resolve(1)]).then(function(r: Arra
 Bluebird.all(['hello', Bluebird.resolve('world'), Promise.resolve('!')]).then(function(r: Array<string>) { });
 Bluebird.all(['hello', Bluebird.resolve('world'), Promise.resolve('!')]).then(function(r: [string, string, string]) { });
 
-// $ExpectError Wrong tuple type
+// $FlowExpectederror Wrong tuple type
```

Comment:
Should these be `$FlowExpectedError` instead of `$FlowExpectederror`?

Reply:
Oh wow you're totally right. I'll look at that.